.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- [ ] As I can see, you did a good job @andrianarivo 😃 , but make sure that there are [no linter errors](https://github.com/microverseinc/linters-config) as it is [required](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Make%20sure%20that%20there%20are%20no%20linter%20errors.). you may check is [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to fix it and keep doing great work 👍

![linters](https://user-images.githubusercontent.com/57408419/226644247-b2ae625f-6f98-4ed9-a947-2fd6a9ef30f4.JPG)